src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const centerWallWidth = wall.clientWidth / 2;
+const centerWallHeight = wall.clientHeight / 2;
+
+const spider = document.querySelector('.spider');
+const centerSpiderWidth = spider.clientWidth / 2;
+const centerSpiderHeight = spider.clientHeight / 2;
+
+spider.style.top = `${centerWallWidth - centerSpiderWidth}px`;
+spider.style.left = `${centerWallHeight - centerSpiderHeight}px`;
```

Comment:
There seems to be a mix-up between the width and height calculations for positioning the spider. The 'top' property should use 'centerWallHeight' and 'centerSpiderHeight', while the 'left' property should use 'centerWallWidth' and 'centerSpiderWidth'. This will correctly center the spider within the wall.